README.md

```diff
@@ -69,4 +79,60 @@ $ curl localhost:8080/metrics
 $ docker build . -t prometheus-sample-app
 $ docker run -it -p 8080:8080 prometheus-sample-app /bin/main -listen_address=0.0.0.0:8080 generate -metric_type=all -is_random=true
 $ curl localhost:8080/metrics
-```
\ No newline at end of file
+```
+
+## Clustering:
+Deploy the example deployment configuration of 5 instances of Prometheus-Sample-App along with configured OTEL Collector.
+    
+### Pre-requisites:
+- Docker
+- Docker Image Prometheus-Sample-App
+- Minikube
```

Comment:
```suggestion
- A Kubernetes cluster
```

This should work on any k8s cluster, not just minikube.

Reply:
@Aneurysm9 thanks. I have updated.